sdk/tables/azure-data-tables/inc/azure/data/tables.hpp

```diff
@@ -12,5 +12,6 @@
 #include "azure/data/tables/dll_import_export.hpp"
 #include "azure/data/tables/models.hpp"
 #include "azure/data/tables/rtti.hpp"
+#include "azure/data/tables/table_sas_builder.hpp"

```

Comment:
Tables plural for consistency?
```suggestion
#include "azure/data/tables/tables_sas_builder.hpp"

```